packages/dialog/src/vaadin-dialog-overlay-mixin.js

```diff
@@ -0,0 +1,255 @@
+/**
+ * @license
+ * Copyright (c) 2017 - 2023 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+
+/**
+ * @polymerMixin
+ */
+export const DialogOverlayMixin = (superClass) =>
+  class DialogOverlayMixin extends superClass {
+    static get properties() {
+      return {
+        /**
+         * String used for rendering a dialog title.
+         */
+        headerTitle: {
```

Comment:
These properties are listed in this mixin to enable observers below. 
As this element is internal, I decided not to copy the entire JSDoc.